### PR TITLE
Update release notes

### PR DESCRIPTION
**Pull request by farshidz**

### CODE REVIEW

**Review by wanliAlex — APPROVED** [review]

File: RELEASE.md [hunk 1]
```diff
@@ -1,3 +1,29 @@
+# Release 2.2.1
+
+## Bug fixes and minor changes
+- Fix response code for vector store timeout, change it from 429 to 504 (https://github.com/marqo-ai/marqo/pull/763)
+
+# Release 2.2.0
+
+## New features
+* Support filtering on document ID with structured indexes. This was already supported with unstructured indexes ([#749](https://github.com/marqo-ai/marqo/pull/749))
+* New structured index data types: `long`, `double`, `array<long>` and `array<double>` for a higher precision and range of values Available for indexes created with Marqo 2.2+ ([#722](https://github.com/marqo-ai/marqo/pull/722))
```
Comment (wanliAlex) [inline RELEASE.md]: range of values. `.` is missing

Comment (wanliAlex) [reply RELEASE.md]: Or maybe change that "... range of values, that are available..."

File: RELEASE.md [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+* Higher precision numeric fields for unstructured indexes. Unstructured indexes created with Marqo 2.2+ will use double precision floats and longs for a higher precision and wider range of values ([#722](https://github.com/marqo-ai/marqo/pull/722))
```
Comment (wanliAlex) [inline RELEASE.md:11]: Maybe `use double precision floats and longs` -> `use double precision floats and long precision integers`  ?

**Review by wanliAlex — CHANGES_REQUESTED** [review]

File: RELEASE.md (hunk 2 above)
Comment (farshidz) [reply RELEASE.md:11]: Unfortunately I can't rephrase things as these notes have been published already. I just had forgotten to add them here in the repo

**Comment by farshidz** [comment]
Changes already merged in

**Pull request closed without merging**